installation/scripts/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
```

Comment:
Hmm, can we simplify it? I would rather move it to the root of the repository, and name it something like `install-minikube.sh`